src/components/organisms/triggers/formParts/scheduler.tsx

```diff
@@ -25,17 +26,32 @@ export const SchedulerFields = () => {
 	}, [cronValue, setValue]);
 
 	return (
-		<div className="relative">
-			<Input
-				aria-label={t("placeholders.cron")}
-				{...register("cron")}
-				isError={!!errors.cron}
-				isRequired
-				label={t("placeholders.cron")}
-				value={cronValue}
-			/>
-
-			<ErrorMessage>{errors.cron?.message as string}</ErrorMessage>
+		<div className="flex flex-col gap-6">
+			<div className="relative">
+				<Input
+					aria-label={t("placeholders.cron")}
+					{...register("cron")}
+					isError={!!errors.cron}
+					isRequired
+					label={t("placeholders.cron")}
+					value={cronValue}
+				/>
+
+				<ErrorMessage>{errors.cron?.message as string}</ErrorMessage>
```

Comment:
Type casting suggests TypeScript types aren't properly configured. Should use proper typing without casting.